components/header/__snapshots__/index.test.jsx.snap

```diff
@@ -45,132 +46,85 @@ exports[`Header with user logged in renders a header 1`] = `
       class="PodBrowser-main-nav-container"
     >
       <nav
-        class="PodBrowser-header-banner__main-nav"
+        class="PodBrowser-main-nav"
+        data-testid="main-nav"
       >
         <ul
           class="PodBrowser-main-nav__list"
         >
           <li
             class="PodBrowser-main-nav__item"
-            data-testid="main-nav-item"
           >
             <a
-              class="PodBrowser-header-banner__aside-menu-trigger"
+              class="PodBrowser-main-nav__link"
+              data-testid="main-menu-files"
               href="/"
-              type="button"
             >
-              <i
-                class="PodBrowser-icon-files PodBrowser-header-banner__aside-menu-trigger-icon"
+              <span
+                class="PodBrowser-icon-files PodBrowser-main-nav__icon"
               />
-              Files
+              <span
+                class="PodBrowser-main-nav__label"
+              >
+                Files
+              </span>
             </a>
           </li>
           <li
             class="PodBrowser-main-nav__item"
-            data-testid="main-nav-item"
           >
             <a
-              class="PodBrowser-header-banner__aside-menu-trigger"
+              class="PodBrowser-main-nav__link"
+              data-testid="main-menu-contacts"
               href="/contacts"
-              type="button"
             >
-              <i
-                class="PodBrowser-icon-users PodBrowser-header-banner__aside-menu-trigger-icon"
+              <span
+                class="PodBrowser-icon-address-book PodBrowser-main-nav__icon"
               />
-              Contacts
+              <span
+                class="PodBrowser-main-nav__label"
+              >
+                Contacts
+              </span>
             </a>
           </li>
           <li
             class="PodBrowser-main-nav__item"
-            data-testid="main-nav-item"
           >
             <a
-              class="PodBrowser-header-banner__aside-menu-trigger"
+              class="PodBrowser-main-nav__link"
+              data-testid="main-menu-bookmarks"
               href="/bookmarks"
-              type="button"
             >
-              <i
-                class="PodBrowser-icon-star PodBrowser-header-banner__aside-menu-trigger-icon"
+              <span
+                class="PodBrowser-icon-star PodBrowser-main-nav__icon"
               />
-              Bookmarks
+              <span
+                class="PodBrowser-main-nav__label"
+              >
+                Bookmarks
+              </span>
             </a>
           </li>
         </ul>
       </nav>
     </div>
     <div
-      class="PodBrowser-userMenu"
+      class="PodBrowser-loading-indicator-container PodBrowser-loading-indicator-container--center"
     >
-      <div
-        class="PodBrowser-header-banner__aside-menu PodBrowser-header-banner__aside-menu--popup"
+      <svg
```

Comment:
Should this snapshot contain the component instead of the spinner?

Reply:
Very good catch! I'd forgotten to include a dependency that the UserMenu component had (`useAuthenticatedProfile`). I've added some more safe-guarding by using data-testids, hopefully that should make the tests more sturdy.

(And as a sidenote: This is an example of snapshots helping us quality-testing our code, happy with that ^_^ )